doc/commands/ddl.md

```
@@ -36,3 +38,46 @@ Find objects that reference the given CDS view
+## apistate list
+
+List API release states for the given CDS view. Displays the release state,
+description, and authorization default value for each contract level (C0-C4).
+
+```bash
+sapcli ddl apistate list ZCDS1
+```
+
+Sample output:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)